test/e2e/storage_performant_policy.go

```diff
@@ -0,0 +1,289 @@
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	v1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/tools/clientcmd/api"
+
+	g "github.com/onsi/ginkgo/v2"
+)
+
+const (
+	clientName         = "cluster-storage-operator-e2e"
+	storageCOName      = "storage"
+	fsGroupPolicyLabel = "storage.openshift.io/fsgroup-change-policy"
+	selinuxPolicyLabel = "storage.openshift.io/selinux-change-policy"
+)
+
+var (
+	waitPollInterval = time.Second
+)
+
+// newClientConfigForTest returns a config configured to connect to the api server
+func newClientConfigForTest() (*rest.Config, error) {
+	loader := clientcmd.NewDefaultClientConfigLoadingRules()
+	clientConfig := clientcmd.NewNonInteractiveDeferredLoadingClientConfig(loader, &clientcmd.ConfigOverrides{ClusterInfo: api.Cluster{InsecureSkipTLSVerify: true}})
+	config, err := clientConfig.ClientConfig()
+	if err == nil {
+		g.GinkgoLogr.Info("Found configuration for", "host", config.Host)
+	}
+	return config, err
+}
+
+var _ = g.Describe("[sig-storage][OCPFeatureGate:StoragePerformantPolicy] Storage Performant Policy", g.Label("Conformance"), g.Label("Parallel"), func() {
+	var kubeClient *kubernetes.Clientset
+
+	g.BeforeEach(func() {
+		kubeConfig, err := newClientConfigForTest()
+		if err != nil {
+			g.Fail(fmt.Sprintf("Failed to get kubeconfig: %v", err))
+		}
+		kubeClient = kubernetes.NewForConfigOrDie(rest.AddUserAgent(kubeConfig, clientName))
+	})
+
+	g.Context("namespace has fsgroup change policy label", func() {
+		var nsObj *v1.Namespace
+		g.BeforeEach(func(ctx context.Context) {
+			nsObj = &v1.Namespace{
+				ObjectMeta: metav1.ObjectMeta{
+					GenerateName: "fsgroup-policy-test-",
+					Labels: map[string]string{
+						fsGroupPolicyLabel: string(v1.FSGroupChangeOnRootMismatch),
+					},
+				},
+			}
+			testContext, cancel := context.WithTimeout(ctx, 5*time.Minute)
```

Comment:
5 minutes should be a const, it's used many times.

Reply:
fixed.